MIGRATING-0.6-0.7.md

```diff
@@ -181,25 +231,129 @@ All `draw_*` methods to draw specific primitives (`draw_circle`, `draw_triangle`
 
 - `clear`
 
-  Fill the entire display with a solid color.
+  Fills the entire display with a solid color.
 
 These methods aim to be more compatible with hardware-accelerated drawing commands. Where possible, embedded-graphics primitives will use `fill_contiguous` and `fill_solid` to improve performance, however may fall back to `draw_iter` by default.
 
-To reduce duplication, please search the `DrawTarget` documentation on <https://docs.rs/embedded-graphics> for more details on the usage and arguments of the above methods.
+To reduce duplication, please search the `DrawTarget` documentation on <https://docs.rs/embedded-graphics-core> for more details on the usage and arguments of the above methods.
 
 ## For crates that handle images
 
-Crates that handle images should now implement items exported by the [`embedded-graphics-core`](https://crates.io/crates/embedded-graphics-core) crate to integrate with embedded-graphics.
+Crates that handle images should now implement the `ImageDrawable` and `OriginDimensions` traits to integrate with embedded-graphics.
 
-The `ImageDrawable` trait has moved there, as well as common use items like the `Dimensions` trait and `Rectangle` primitive.
+The below examples shows an implementation for an imaginary `MyRgb888Image` which uses 24 bit color and draws to targets that support the same.
 
-TODO: Improve this section before release.
+```rust
+use embedded_graphics::{
+    draw_target::{DrawTarget, DrawTargetExt},
+    geometry::{OriginDimensions, Size},
+    image::ImageDrawable,
+    pixelcolor::{PixelColor, Rgb888},
+    primitives::Rectangle,
+};
+
+struct MyRgb888Image {
+  // ...
+}
+
+impl ImageDrawable<Rgb888> for MyRgb888Image {
+    type Color = Rgb888;
+
+    fn draw<D>(&self, target: &mut D) -> Result<(), D::Error>
+    where
+        D: DrawTarget<Color = Rgb888>,
+    {
+        // ... send contiguous pixels read from the image to the target ...
+    }
+
+    fn draw_sub_image<D>(&self, target: &mut D, area: &Rectangle) -> Result<(), D::Error>
+    where
+        D: DrawTarget<Color = Self::Color>,
+    {
+        // Delegate to the draw() method using a reduced draw target
+        self.draw(&mut target.translated(-area.top_left).clipped(area))
+    }
+}
+
+impl OriginDimensions for MyRgb888Image {
+    fn size(&self) -> Size {
+        // Return image width and height in pixels
+    }
+}
+```
 
 ## For text rendering crates
 
-Crates that handle text rendering should now implement items exported by the [`embedded-graphics-core`](https://crates.io/crates/embedded-graphics-core) crate to integrate with embedded-graphics.
+### Monospace fonts
```

Comment:
This section needs to be updated when #596 gets merged.

Reply:
I've updated the section. Let me know what you think.